addon/utils/cp/size-class.js

```diff
@@ -0,0 +1,12 @@
+import { computed } from '@ember/object';
+import { isBlank } from '@ember/utils';
+import { assert } from '@ember/debug';
+
+export default function sizeClass(prefix) {
+  assert('You have to provide a prefix for sizeClass', prefix);
+
+  return computed('size', function() {
+    let size = this.get('size');
+    return isBlank(size) ? null : `${prefix}-${size}`;
```

Comment:
We may want to add an assertion that the value of size is valid. At least that it's a string.

Reply:
ok.